.github/CONTRIBUTING.md

```diff
@@ -780,18 +780,27 @@ Remember that unit tests test a single unit - so if you find yourself relying on
 
 Being able to read a test as a sentence, makes it clear what has gone wrong when the test fails and makes it act as a functional requirement. 
 
-In the following two examples, note how example #2 clearly communicates intent while being easy to read and understand.
+In the following two examples - note how the _good example_ clearly communicates intent while being easy to read and understand.
 
-Example #1 
+Consider the following _bad_ example: 
```

Comment:
This is a bit messy...
Maybe something along the lines of: "Consider the following two examples. In the first example... In the second example notice how..."

Reply:
Took another stab at it. Not exactly what you proposed, but something along the lines of that 😄